spec/models/cart_spec.rb

```diff
@@ -2,23 +2,23 @@
 
 RSpec.describe Cart, type: :model do
   it { is_expected.to belong_to(:user) }
-  it { is_expected.to have_many(:order_items) }
+  it { is_expected.to have_many(:line_items) }
 
   describe "#calculate_total" do
     let(:cart) { create(:cart) }
-    let(:item_1) { create(:order_item, cart: cart) }
-    let(:item_2) { create(:order_item, cart: cart) }
+    let(:item_1) { create(:line_item, cart: cart) }
+    let(:item_2) { create(:line_item, cart: cart) }
     it "calculates the total price for the order" do
-      cart.order_items << item_1
-      cart.order_items << item_2
+      cart.line_items << item_1
+      cart.line_items << item_2
```

Comment:
You shouldn't need to do these two lines. I'm pretty sure they happen automatically from adding `cart: cart` on lines 9 and 10.